vuepress/docs/v6.3.2/docs/getting-started/README.md

```diff
@@ -32,19 +52,18 @@ curl -sfL https://get.entando.org | bash
 
 ## Manual Install
 
-This in-depth guide takes a learn-as-you-go approach. It will give you a working knowledge of Kubernetes as you get Entando up and running in a local environment.
+Configure a shared cluster or customize a local developer environment. This is a learn-as-you-go approach. It will give you a working knowledge of Kubernetes as you get Entando up and running in a local environment. The process is as follows:
 
 1. [Install Kubernetes](#install-kubernetes)
 2. [Prepare Kubernetes Environment](#prepare-kubernetes)
 3. [Deploy Entando](#deploy-entando)
 
-Note: For advanced or long-time Entando users, check out our [Quick Reference](quick-reference) install guide for just the steps.
-
-Since Entando is designed to run on Kubernetes, let's get started by installing our own instance of Kubernetes locally.
+We've tested a variety of Kubernetes implementations, including Minikube, Minishift, CodeReady Containers, K3s, and Microk8s, to find the optimal combination of low cpu/memory usage, fast startup times, and minimal configuration. After downloading the necessary files, you'll have your own instance of Kubernetes up and running in <60 seconds.
 
-We've tested a variety of Kubernetes implementations, including Minikube, Minishift, CodeReady Containers, K3s, and Microk8s, to find the optimal combination of low cpu/memory usage, fast startup times, and minimal configuration. After downloading the necessary files, we'll have our own instance of Kubernetes up and running in <60 seconds.
+::: tip Notes:
+For advanced or long-time Entando users, check out our [Quick Reference](quick-reference) install guide for just the steps.
 
-::: tip What's Needed to Run Kubernetes?
+**What's Needed to Run Kubernetes?**
```

Comment:
Minor, but I think it looked better as separate tips? They aren't actually related tips.
![image](https://user-images.githubusercontent.com/645546/142495405-8a31277e-04da-4f78-93be-d9f3f3093178.png)